scripts/scil_compute_todi.py

```diff
@@ -33,39 +35,39 @@ def _build_arg_parser():
     p = argparse.ArgumentParser(description=__doc__, epilog=EPILOG,
                                 formatter_class=argparse.RawTextHelpFormatter)
 
-    p.add_argument('in_bundle',
+    p.add_argument('input_tracto',
                    help='Input streamlines file.')
 
     add_reference_arg(p)
 
     p.add_argument('--sphere', default='repulsion724',
-                   help='sphere used for the angular discretization.')
+                   help='sphere used for the angular discretization. '
+                        '[%(default)s]')
 
     p.add_argument('--mask',
                    help='Use the given mask.')
 
     p.add_argument('--out_mask',
                    help='Mask showing where TDI > 0.')
 
-    p.add_argument('--out_lw_tdi',
-                   help='Output length-weighted TDI map.')
+    p.add_argument('--out_tdi',
+                   help='Output Track Density Image (TDI).')
 
-    p.add_argument('--out_lw_todi',
-                   help='Output length-weighted TODI map.')
+    p.add_argument('--out_todi_sf',
+                   help='Output TODI, with SF (each directions\n'
+                        'on the sphere), require a lot of memory)')
```

Comment:
The parenthesis seems to not fit.

Reply:
remove ) after sphere

require -> requires

Reply:
done